cypress/integration/checkout/checkout.multipiece.items.spec.js

```diff
@@ -0,0 +1,152 @@
+/// <reference types="cypress" />
+
+import TestTypes from '../../support/dictionary/testTypes';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import getRandomPostfix from '../../support/utils/stringTools';
+import Helper from '../../support/fragments/finance/financeHelper';
+import CheckOutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import NewServicePoint from '../../support/fragments/service_point/newServicePoint';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints';
+import ConfirmMultipieceCheckOutModal from '../../support/fragments/checkout/confirmMultipieceCheckOutModal';
+import SwitchServicePoint from '../../support/fragments/service_point/switchServicePoint';
+
+describe('Check Out', () => {
+  let user = {};
+  const FIRST_ITEM_BARCODE = Helper.getRandomBarcode();
+  const SECOND_ITEM_BARCODE = Helper.getRandomBarcode();
+  const THIRD_ITEM_BARCODE = Helper.getRandomBarcode();
+  const FOURTH_ITEM_BARCODE = Helper.getRandomBarcode();
+  let userBarcode = '';
+  const instanceTitle = `autotest_instance_title_${getRandomPostfix()}`;
+  const quantityPiecesForSecondItem = '3';
+  const quantityPiecesForThirdItem = '2';
+  const quantityPiecesForFourthItem = '1';
+  const quantityOfMissingPieces = '2';
+  const descriptionOfPiece = `autotest_description_${getRandomPostfix()}`;
+  const missingPieceDescription = `autotest_description_${getRandomPostfix()}`;
+  let servicePoint;
+
+  beforeEach(() => {
+    cy.createTempUser([
+      permissions.checkoutCirculatingItems.gui,
+      permissions.uiUsersView.gui,
```

Comment:
add real list of permissions into testrail TC

Reply:
permission 'uiUsersView' is used in only afterEach block, that's why it is not related to test case